Sources/Commands/SwiftPackageTool.swift

```diff
@@ -60,7 +60,9 @@ public struct SwiftPackageTool: ParsableCommand {
             ComputeChecksum.self,
             ArchiveSource.self,
             CompletionTool.self,
-        ] + (ProcessInfo.processInfo.environment["SWIFTPM_ENABLE_SNIPPETS"] == "1" ? [Learn.self] : [ParsableCommand.Type]()),
+        ]
+        + (ProcessInfo.processInfo.environment["SWIFTPM_ENABLE_SNIPPETS"] == "1" ? [Learn.self] : [])
```

Comment:
Took the liberty of simplifying this line since the compiler seems to have no problem inferring the type of the `[]` for the `:` case.